modules/deploy-red-hat-openshift-container-storage.adoc

```diff
@@ -9,42 +9,51 @@
+|link:https://access.redhat.com/documentation/en-us/red_hat_openshift_data_foundation/4.9/html/deploying_openshift_data_foundation_using_microsoft_azure_and_azure_red_hat_openshift[Deploying and managing OpenShift Data Foundation 4.9 using Microsoft Azure]
+
+|Allocating storage to core services and hosted applications in {rh-storage-first}, including snapshot and clone
+|link:https://access.redhat.com/documentation/en-us/red_hat_openshift_data_foundation/4.9/html/managing_and_allocating_storage_resources[Managing and allocating resources]
+
+|Managing storage resources across a hybrid cloud or multicloud environment using the Multicloud Object Gateway (NooBaa)
+|link:https://access.redhat.com/documentation/en-us/red_hat_openshift_data_foundation/4.9/html/managing_hybrid_and_multicloud_resources[Managing hybrid and multicloud resources]
+
+|Safely replacing storage devices for {rh-storage-first}
+|link:https://access.redhat.com/documentation/en-us/red_hat_openshift_data_foundation/4.9/html/replacing_devices[Replacing devices]
+
+|Safely replacing a node in a {rh-storage-first} cluster
+|link:https://access.redhat.com/documentation/en-us/red_hat_openshift_data_foundation/4.9/html/replacing_nodes[Replacing nodes]
 
-|Managing a {rh-storage-first} 4.5 cluster
-|link:https://access.redhat.com/documentation/en-us/red_hat_openshift_container_storage/4.5/html/managing_openshift_container_storage/index[Managing OpenShift Data Foundation 4.5]
+|Scaling operations in {rh-storage-first}
+|link:https://access.redhat.com/documentation/en-us/red_hat_openshift_data_foundation/4.9/html/scaling_storage[Scaling storage]
 
-|Monitoring a {rh-storage-first} 4.5 cluster
-|link:https://access.redhat.com/documentation/en-us/red_hat_openshift_container_storage/4.5/html/monitoring_openshift_container_storage/index[Monitoring Red Hat OpenShift Data Foundation 4.5]
+|Monitoring a {rh-storage-first} 4.9 cluster
+|link:https://access.redhat.com/documentation/en-us/red_hat_openshift_data_foundation/4.9/html/monitoring_openshift_data_foundation[Monitoring Red Hat OpenShift Data Foundation 4.9]
 
 |Resolve issues encountered during operations
-|link:https://access.redhat.com/documentation/en-us/red_hat_openshift_container_storage/4.5/html/troubleshooting_openshift_container_storage/index[Troubleshooting OpenShift Data Foundation 4.5]
+|link:https://access.redhat.com/documentation/en-us/red_hat_openshift_data_foundation/4.9/html/troubleshooting_openshift_data_foundation[Troubleshooting OpenShift Data Foundation 4.9]
 
 |Migrating your {product-title} cluster from version 3 to version 4
-|link:https://access.redhat.com/documentation/en-us/openshift_container_platform/4.7/html/migrating_from_version_3_to_4/index[Migration]
+|link:https://access.redhat.com/documentation/en-us/openshift_container_platform/4.9/html/migration_toolkit_for_containers/ersion_3_to_4/index[Migration]
```

Comment:
This link has a typo (should be version) but even in fixing that, it still shows 404. Should this link be `https://access.redhat.com/documentation/en-us/openshift_container_platform/4.9/html/migrating_from_version_3_to_4/index`?